src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+    add(val, res) {
+      return res + val;
+    },
+
+    subtract(val, res) {
+      return res - val;
+    },
+
+    multiply(val, res) {
+      return res * val;
+    },
+
+    divide(val, res) {
+      return res / val;
+    },
```

Comment:
You don't need to expect two parameters here. You can reference the result from inside the method.